package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hof-template-mixins",
-  "version": "5.2.0",
+  "version": "5.2.1",
```

Comment:
I'd suggest this is a new feature and that this should be a bump to the minor version rather than patch.

Reply:
Oh sorry is this a bug fix on your prior work?

In that case this is fine.

Reply:
Just saw you clarification in the main comment thread.

Reply:
@daniel-ac-martin it is, I finally figured out how to run a hof project in docker with locally amended modules - so could test it this time!